src/components/utils/utils.ts

```diff
@@ -0,0 +1,90 @@
+import html2canvas from "html2canvas";
+
+export const download = async (id: string): Promise<void> => {
```

Comment:
function name should be more meaningful even if it gets longer.
```suggestion
export const downloadImageFromElementId = async (id: string): Promise<void> => {
```